week06/manager_test.go

```diff
@@ -0,0 +1,162 @@
+package week06
+
+import (
+	"testing"
+)
+
+func Test_Manager_Start_Fail(t *testing.T) {
+	m := NewManager()
+	exp := ErrInvalidEmployeeCount(0)
+
+	act := m.Start(0)
+
+	if act.Error() != exp.Error() {
+		t.Fatalf("expected %q got %q", exp.Error(), act.Error())
+	}
+}
+
+func Test_Manager_Start_Success(t *testing.T) {
+	m := NewManager()
```

Comment:
You need to make sure you are cleaning up your resources. Currently, your test is leaking resources because you haven't called `m.Stop` at any point. You should immediately defer a call to `m.Stop` to ensure that regardless of your tests exit, you properly clean up after yourself.

```suggestion
	m := NewManager()
	defer m.Stop()
```